<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>XIVN1987</author>
 <class>SVDView</class>
 <widget class="QWidget" name="SVDView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XIVN1987 SVD Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Image/serial.ico</normaloff>Image/serial.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout0">
   <item>
    <layout class="QHBoxLayout" name="hLayout1">
     <item>
      <widget class="QLabel" name="lblDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>JLinkARM.dll：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linDLL"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout2">
     <item>
      <widget class="QLabel" name="lblSVD">
       <property name="text">
        <string>SVD 文件路径：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbSVD">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSVD">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout3">
     <item>
      <widget class="QLabel" name="lblPeriph">
       <property name="text">
        <string>MCU 外设列表：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbPeriph">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="linPeriph">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree">
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="headerVisible">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">偏移</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">描述</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
